### DISCUSSION TITLE
Support for SSD1322 OLEDs? (e.g. NHD-2.7-12864WD)

### DISCUSSION THREAD

**Discussion by DatanoiseTV** [Bodmer/TFT_eSPI#2415] [New processor/display support or capability request]
I do own a few older NHD-2.7-12864WD displays which are using the SSD1322 controller.
Is there any plan to support these screens as well? They have an excellent contrast and supports greyscale.

Controller Datasheet: https://newhavendisplay.com/content/app_notes/SSD1322.pdf

They can be driven in Parallel Mode or SPI.

**Comment by Bodmer** [comment]
It is unlikely that support will be added. You may find other libraries that support that controller.

**Reply by DatanoiseTV** [reply]
Ok, I was hoping TFT_eSPI would be supporting it in the future. There is unfortunately no driver which uses DMA or which is fast.

**Reply by Bodmer** [reply]
I expect the U8g2 library would give adequate performance without DMA, but it is a library I have used.
